1_data/src/pull_nwis.R

```diff
@@ -0,0 +1,17 @@
+
+pull_nwis <- function(ind_file, sites_yml, dates_yml, params_yml, gd_config) {
+
+  dates <- yaml::yaml.load_file(dates_yml)
+  pcodes <- yaml::yaml.load_file(params_yml)
+  site_ids <- yaml::yaml.load_file(sites_yml)
+
+  flow <- readNWISuv(siteNumbers=site_ids, parameterCd=pcodes$flow, startDate=dates$calibrate$start, endDate=dates$forecast$start)
+  nitrate <- readNWISuv(siteNumbers=site_ids, parameterCd=pcodes$nitrate, startDate=dates$calibrate$start, endDate=dates$forecast$start)
+
+  writeLines(list(flow=flow, nitrate=nitrate), outfile)
```

Comment:
let's delete this one b/c it's duplicated below (and uses `outfile` which no longer exists in thix context)

Reply:
oops good catch